ansible/group_vars/all/vars.yml

```diff
@@ -47,3 +48,4 @@ passwd:
 ansible_venv: /root/venv
 docker_apt_ver: ""
 airflow_tag: "1.8.0-oo3"
+oomsm_tag: "1.0.0-beta.1"
```

Comment:
BTW, I'm moving it to role/vars instead of role/defaults as I was unaware of the difference between role/vars and role/defaults and I think that I should review all the stuff I've placed to role/vars here and there :)